atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/TwoNodesDownTableManipulationTest.java

```diff
@@ -31,35 +29,10 @@ void testSetup(CassandraKeyValueService kvs) {
     }
 
     @Test
-    public void createTableThrowsAndDoesNotChangeCassandraSchema() throws TException {
+    public void createTableThrowsAndDoesNotChangeCassandraSchema() {
         TableReference tableToCreate = TableReference.createWithEmptyNamespace("new_table");
         assertThrowsAtlasDbDependencyExceptionAndDoesNotChangeCassandraSchema(() ->
                 getTestKvs().createTable(tableToCreate, AtlasDbConstants.GENERIC_TABLE_METADATA));
 
     }
-
-    @Test
-    public void dropTableThrowsAndDoesNotChangeCassandraSchema() throws TException {
-        assertThrowsAtlasDbDependencyExceptionAndDoesNotChangeCassandraSchema(() ->
-                getTestKvs().dropTable(TEST_TABLE));
-    }
-
-    @Test
-    public void dropTablesThrowsAndDoesNotChangeCassandraSchema() throws TException {
-        assertThrowsAtlasDbDependencyExceptionAndDoesNotChangeCassandraSchema(() ->
-                getTestKvs().dropTables(ImmutableSet.of(TEST_TABLE)));
-    }
-
-    @Test
-    public void truncateTableThrowsAndDoesNotChangeCassandraSchema() {
-        assertThrowsAtlasDbDependencyExceptionAndDoesNotChangeCassandraSchema(() ->
-                getTestKvs().truncateTable(TEST_TABLE));
-
-    }
-
-    @Test
-    public void truncateTablesThrowsAndDoesNotChangeCassandraSchema() {
-        assertThrowsAtlasDbDependencyExceptionAndDoesNotChangeCassandraSchema(() ->
-                getTestKvs().truncateTables(ImmutableSet.of(TEST_TABLE)));
-    }
```

Comment:
I guess the rationale for removing these tests here is "if we can't do it with 1 node down, we almost certainly can't do it with 2 down"?